Automattic/woocommerce-payments
# update: show LPM icon on order success page

Fixes https://github.com/Automattic/woocommerce-payments/issues/9682

#### Changes proposed in this Pull Request









Adding support for LPMs on "order success" pages.
Google Pay/Apple Pay/Link/Cards remain unaffected.

Here are some screenshots for proof:
<details><summary>Details</summary>
<p>
<img width="824" alt="Screenshot 2024-11-07 at 12 55 58 PM" src="https://github.com/user-attachments/assets/e363de4f-f0d0-4d6a-a18a-a500d46fa62d">
<img width="833" alt="Screenshot 2024-11-07 at 12 55 49 PM" src="https://github.com/user-attachments/assets/1b36e435-a858-4312-bcdc-62fb9784864f">
<img width="817" alt="Screenshot 2024-11-07 at 12 55 43 PM" src="https://github.com/user-attachments/assets/48e0fca8-7aa6-4d74-88a7-7ae6a620ec76">
<img width="813" alt="Screenshot 2024-11-07 at 12 55 36 PM" src="https://github.com/user-attachments/assets/b07cf6ff-65a2-4ad8-bd27-d3e5947825a6">
<img width="838" alt="Screenshot 2024-11-07 at 12 55 30 PM" src="https://github.com/user-attachments/assets/b7f0dbd0-6b8b-44cc-b280-a756f33387c2">
<img width="822" alt="Screenshot 2024-11-07 at 12 55 23 PM" src="https://github.com/user-attachments/assets/f0907e5a-16e7-453c-a116-c252609d7f66">
<img width="828" alt="Screenshot 2024-11-07 at 12 55 11 PM" src="https://github.com/user-attachments/assets/16a5cc1c-7812-4935-8981-739fcad49d73">
<img width="816" alt="Screenshot 2024-11-07 at 12 56 08 PM" src="https://github.com/user-attachments/assets/6fe92658-bc49-4548-9d7a-880b4a91b1fb">
</p>
</details> 

I also ensured to run the full suite of e2e tests: https://github.com/Automattic/woocommerce-payments/actions/runs/11722674600

#### Testing instructions





- Ensure you have LPMs enabled in your environment
- As a customer, add a product to the cart
- You'll need to test this with different scenarios
  - For BNPLs: ensure you have > 50$ worth of products, then checkout
  - For P24: Use PLN currency at checkout, billing address `01-301 Warszawa Poland`
  - For SEPA/EPS/Sofort: Use EUR currency at checkout, billing address `1010 Vienna Austria`
<img width="834" alt="Screenshot 2024-11-07 at 11 42 03 AM" src="https://github.com/user-attachments/assets/466877f5-b0a7-4f8c-96a6-2da95793590a">
- "Order success" page will show the payment method's logo (but it will keep not showing it for GooglePay/ApplePay/Link/Cards)


-------------------


**Post merge**
